scripts/check-pr-comments.test.sh

```
@@ -0,0 +1,180 @@
+cat > "$MOCK_GH" << 'MOCK'
+#!/usr/bin/env bash
+if [[ "$*" == *"pulls"*"comments"* ]]; then
+    echo "Error: API request failed" >&2
+    exit 1
+elif [[ "$*" == *"nameWithOwner"* ]]; then
+    echo "owner/repo"
+fi
+MOCK
+chmod +x "$MOCK_GH"
+
+output=$(PATH="$MOCK_DIR:$PATH" bash "$SCRIPT" --pr 999 --repo owner/repo 2>&1)
```

Shellcheck (0.11.0)

[warning] output appears unused. Verify use (or export if used externally).

(SC2034)